src/main/java/frc/robot/end_effector/EndEffectorConstants.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.end_effector;
+
+public class EndEffectorConstants {
+    
+    public static final int algaeIntakeCAN = 38; //TODO AFTER ROBOT IS WIRED
```

Comment:
When writing TODO comments, explain what it is that needs to be done for better organization.

`// TODO Algae Intake CAN ID`